frontend/src/routes/Governance/common/util.tsx

```diff
@@ -492,19 +494,61 @@ export function getPolicyRemediation(policy: Policy | undefined) {
   }
 
   remediationAggregation = remediationAggregation.trim().replaceAll(' ', '/')
-  if (remediationAggregation === '') {
-    return '-'
+
+  if (rootRA === REMEDIATION_ACTION.ENFORCE) {
+    return remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY) ? rootRA.concat('/informOnly') : rootRA
   }
-  if (policy.spec?.remediationAction) {
-    if (remediationAggregation.includes('informOnly')) {
-      if (remediationSet.size > 1) {
-        return policy.spec.remediationAction.concat('/informOnly')
-      }
-      return 'informOnly'
+
+  if (rootRA) {
+    if (remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY)) {
+      rootRA = remediationSet.size > 1 ? rootRA.concat('/informOnly') : REMEDIATION_ACTION.INFORM_ONLY
+    }
+  } else {
+    rootRA = remediationAggregation
+    // empty root remediation and template remedation all empty
+    if (!rootRA) {
```

Comment:
It'd technically work if you had no remediationAction set on the root policy and policy templates but used an override on all clusters.

Reply:
Yes

Reply:
Can you give me the case you concern?

Reply:
root remediation null and 0 overridden on propagated fall into here